services/web/src/components/ErrorMessage.js

```diff
@@ -0,0 +1,47 @@
+// Displays an error message if an error is passed
+// also captures and logs error stack for production
+// debugging.
+
+import React from 'react';
+import { Message } from 'semantic';
+import { ApiError } from 'utils/api';
+import { CustomError } from 'utils/error';
+import { captureError } from 'utils/sentry';
+
+export default class ErrorMessage extends React.Component {
+  componentDidUpdate(lastProps) {
+    const { error } = this.props;
+    if (error !== lastProps.error) {
+      if (this.canLogError(error)) {
+        // eslint-disable-next-line
+        console?.error(error);
+        captureError(error);
+      }
+      this.setState({
+        open: false,
+      });
+    }
+  }
+
+  canLogError(error) {
+    if (error instanceof ApiError) {
+      return error.status >= 500;
```

Comment:
would that lead to double errors because sentry will hook into fetch ?

Reply:
it might :) ... i'll double check

Reply:
Confirmed with sentry... only seeing one error